setup.rst

```diff
@@ -341,6 +341,14 @@ Now you can install the build dependencies via ``apt``::
 If that package is not available for your system, try reducing the minor
 version until you find a package that is available.
 
+If you want to build all modules, install these packages in the versions you
+prefer, changing the version name as above::
+
+   $ sudo apt-get install build-essential gdb lcov libbz2-dev libffi-dev \
+         libgdbm-dev liblzma-dev libncurses5-dev libreadline6-dev \
+         libsqlite3-dev libssl-dev lzma lzma-dev tk-dev uuid-dev xvfb \
```

Comment:
`xvfb` is not actually necessary, it would only allow you to run Tk tests on a headless machine.

Reply:
I have removed `xvfb` from the list.